README.md

```diff
@@ -57,22 +59,33 @@ When the reader has completed this Code Pattern, they will understand how to:
 * In addition setup your [AWS S3 command line](https://aws.amazon.com/cli/) which can be used to create buckets and/or add files to COS.
    * Export `AWS_ACCESS_KEY_ID` with your COS `access_key_id` and `AWS_SECRET_ACCESS_KEY` with your COS `secret_access_key`
 
-### Setup IBM CLI & ML CLI
+### Provision a Watson Machine Learning service instance
+
+* Log in to the [IBM Cloud](https://console.bluemix.net/). Sign up for a free account if you don't have one yet.
+* [Provision a Watson Machine Learning service instance](https://console.bluemix.net/catalog/services/machine-learning) on IBM Cloud if you don't have one 
+* Create new credentials 
+	* On the left side, click the `service credentials`
+	* Click on the `new credentials` button to create new credentials
+	* In the `Add New Credentials` popup accept the defaults and create the credentials.
+	* View the generated credentials and take note of the values for `instance`, `username`, `password`, and `url`. You will need this information when you configure your IBM Cloud CLI.	
+
+
+### Set up IBM CLI & ML CLI
 
 * Install [IBM Cloud CLI](https://console.bluemix.net/docs/cli/reference/bluemix_cli/get_started.html#getting-started)
-  * Login using `bx login` or `bx login --sso` if within IBM
+  * Login using `ibmcloud login` or `ibmcloud login --sso` if within IBM
 * Install [ML CLI Plugin](https://dataplatform.ibm.com/docs/content/analyze-data/ml_dlaas_environment.html)
   * After install, check if there is any plugins that need update
```

Comment:
I think this sentence needs to be re-written

Reply:
Maybe
```suggestion
  * After installation check for plugin updates
```

Reply:
how about this:

Verify that the latest pkug-in version is installed: `ibmcloud plugin update machine-learning`